src/material/list/list.scss

```diff
@@ -162,15 +162,24 @@ a.mdc-list-item--activated {
   }
 }
 
-.mat-mdc-list-item.mdc-list-item--with-three-lines {
-  // List item lines or titles never wrap. MDC always enables wrapping for secondary text
-  // if the list item has acquired three lines. We unset these styles for line elements.
-  // https://github.com/material-components/material-components-web/blob/348665978ce73694ad4518626dd70cdf5b984113/packages/mdc-list/_evolution-mixins.scss#L205-L206.
-  // TODO: Consider removing once MDC supports the explicit tertiary line list variant.
+.mat-mdc-list-item.mdc-list-item--with-leading-avatar.mdc-list-item--with-two-lines
+.mdc-list-item__primary-text::before {
+  height: 25px;
+}
+
+.mat-mdc-list-item.mdc-list-item--with-three-lines,
+.mat-mdc-list-item.mdc-list-item--with-two-lines {
+  // List item lines must wrap according to GAR 1.18(b). Removing the nowrap and
+  // adjusts .mat-mdc-list-item-title.mdc-list-item__primary-text height
+  // to accommodate any wrapping text. Fixes: b/247881646.
+  .mat-mdc-list-item-title.mdc-list-item__primary-text,
   .mat-mdc-list-item-line.mdc-list-item__secondary-text {
-    white-space: nowrap;
+    white-space: normal;
     line-height: normal;
   }
+  .mat-mdc-list-item-title.mdc-list-item__primary-text {
+    height: 45px;
```

Comment:
Hi @andrewseguin! Removed previous attempt to improve readability and revised by increasing the list items with 3 lines from 88px to min-height 90px so that the bottom letters of the 3rd line do not get cut off. See screenshots for reference:

- Before [screenshot](https://screenshot.googleplex.com/BngZQoohnfCgsaQ)
- After [screenshot](https://screenshot.googleplex.com/4LPS2RyTVeWUUqe)

Reply:
Hardcoding the heights won't work if the text needs to wrap to several lines. Try adding more secondary text to see that it only shows two lines.

Ideally the text container grows as necessary and doesn't set height at all. This might require making more significant changes to the list item styles

Reply:
Hi @andrewseguin! Thanks for the rec. I did some further digging and it looks like that variable (value of 88px) is coming from _list.js theming for density (screenshot [here](https://screenshot.googleplex.com/6j7GQ29bSzejHtP)) . Would it be better to override the value by adding in a `height: auto` in the `list.scss` or remove the specific values from the theming? Honestly I haven't worked directly with affecting overall theming so wanted to check before moving in that direction.